test/built-ins/Temporal/PlainDateTime/from/argument-number.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.from
+description: A number might be an acceptable argument
```

Comment:
I'd describe this one as "A number is converted to a string, which may be a valid ISO string"

Reply:
or "A number is converted to a string, then to Temporal.PlainDateTime"

Reply:
sure, sounds good

done